Cargo.toml

```diff
@@ -44,7 +44,7 @@ libc = "0.2"
 nix = "0.26"
 bitflags = "1.3"
 thiserror = "1.0"
-socket2 = { version = "0.5", features = ["all"] }
+socket2 = { git = "https://github.com/rust-lang/socket2.git", features = ["all"] }
```

Comment:
Support for `MSG_CONFIRM` was just added (https://github.com/rust-lang/socket2/commit/c93cdcc25fd3de7ef20515b9c16ed2b5771ed268). Probably best to wait for a new release before this PR can be merged.